spec/features/daily_digest_spec.rb

```diff
@@ -36,7 +36,7 @@
         description: "Description one",
         change_note: "Change note one",
         public_updated_at: "2017-01-01 10:00:00",
-        links: { topics: [list_one_topic_id] },
+        links: { taxons: [list_one_taxon_id] },
```

Comment:
Just checked in the db and ContentChange uses both the key `taxons` and `taxon_tree`. No idea why! But just letting you know that it's fine to use `taxons` here.